internal/version/version.go

```diff
@@ -0,0 +1,11 @@
+// Do not modify this file manually. It is modified by the release script.
+package version
+
+// DDLambdaVersion is the current version number of this library (datadog-lambda-go).
+// It is automatically updated by the release script.
+const DDLambdaVersion = "0.10.0"
```

Comment:
Have you looked at `runtime/debug.ReadBuildInfo` to try to inspect the `DDLambdaVersion` and `DDTraceVersion` automatically at runtime? https://golang.org/pkg/runtime/debug/#ReadBuildInfo

This is a good solution, but it might be a little cleaner.

Reply:
That might work for the DD Trace version, but would it work for the DD Lambda version? That is the version number of this library itself -- afaik it's not encoded anywhere in the module, it's just something in GitHub.

Reply:
On one hand, the user is importing our library at a specific version so the Go runtime should be able to know what version has been imported (my guess is it'll be listed among the `Deps`). On the other hand, I haven't proven that to myself yet. It might be a boondoggle to get working correctly.

Reply:
I think I'll just stick to this implementation for now -- I think it will work and I'd rather not spend the time to re-implement it. But I like your idea though!